database-seeds/scripts/test/id.test.ts

```diff
@@ -117,6 +116,14 @@ for (const [collection, uniqueIDs] of Object.entries(UniqueKeys)) {
 			if (Array.isArray(uniqueID)) {
 				const mappedProps = uniqueID.map((e) => get(d, e));
 
+				// Charts are special and can be a duplicate if it's not primary
+				if(collection.startsWith("charts-")) {
+					if(mappedProps[uniqueID.lastIndexOf("isPrimary")] == false ) {
```

Comment:
use `===` and run eslint on this file

logic here is insane but i get it,,

Reply:
Done